src/BTVNanoCommissioning/helpers/update_branch.py

```diff
@@ -32,7 +32,7 @@ def missing_branch(events):
                     (events.Jet.btagDeepFlavC / (1.0 - events.Jet.btagDeepFlavB)),
                     -1,
                 ),
-                0.999999,
+                0.999_999,
```

Comment:
this number doesn't look right to me (i.e. the `_` within the number (4 times in total)